misc/menu_templates/main.in.hjson

```diff
@@ -178,6 +178,19 @@
                         }
                     ]
                 }
+                {
+                    value: { command: "SSH" }
+                    action: [
+                        {
+                            //  Allow SSH key management only from secure transports
+                            acs: SC
+                            action: @menu:userSSHKeyConfig
```

Comment:
One thing I did with the WFC menu was to force a couple of ACS. Ops can add additional restrictions, but they always get `SC`: https://github.com/NuSkooler/enigma-bbs/blob/master/core/wfc.js#L68